shared/AppInsightsCommon/src/Telemetry/Exception.ts

```diff
@@ -533,6 +533,12 @@ export class _ExceptionDetails implements IExceptionDetails, ISerializable {
             _self.message = dataSanitizeMessage(logger, _formatMessage(exception || error, _self.typeName)) || strNotSpecified;
             const stack = exception[strStackDetails] || _getStackFromErrorObj(exception);
             _self.parsedStack = _parseStack(stack);
+
+            // after parsedStack is inited, iterate over each frame object, sanitize its assembly field
+            if (_self.parsedStack instanceof Array){
```

Comment:
We have an `isArray` helper function that you can use here.
But, yes this looks like it would address the raised issue.
Please go ahead and create a test case for this